language-server/src/server.js

```diff
@@ -304,4 +310,39 @@ connection.languages.semanticTokens.onDelta(({ textDocument, previousResultId })
   return builder.buildEdits();
 });
 
+// $SCHEMA COMPLETION
+function findPropertyAtPosition(instance, position) {
+  for (const [key, value] of instance.entries()) {
+    if (
+      position.line >= key.startPosition().line
+      && position.character >= key.startPosition().character
+      && position.line <= value.endPosition().line
+      && position.character <= value.endPosition().character
+    ) {
+      return { key, value };
+    }
+  }
+  return undefined;
+}
```

Comment:
I think this would be useful as a method in JsoncInstance instead of as a function by itself.

Can you use `findNodeAtOffset` in [jsonc-parser](https://www.npmjs.com/package/jsonc-parser) to avoid some work here?

Reply:
Yeah, we can avoid some work in the method by using `findNodeAtOffset`.